File: UVa-Online-Judge/11799 - Horror Dash.py
# ==================================================
# Problem  :   11799 - Horror Dash
# Run time :   0.010 sec.
# Language :   Python 3
# ==================================================

def main():
	from sys import stdin, stdout

	# stdin = open('input.txt', 'r')

	inp = [int(x) for x in stdin.read().split()]

	t = inp.pop(0)
	a, b = 0, 0

	for tc in range(1, t+1):
		n = inp[a]

		a += 1
		b = a + n
		ans = max(inp[a:b])
		a = b

		stdout.write("Case %d: %d\n" % (tc, ans))

if __name__ == '__main__':
	main()

File: UVa-Online-Judge/11799 - Horror Dash.c
// ==================================================
// Problem  :   11799 - Horror Dash
// Run time :   0.000 sec.
// Language :   C
// ==================================================

#include <stdio.h>

int main()
{
    int t, tc, n, c, mx;
    scanf("%d", &t);

    for(tc = 1; tc <= t; ++tc) {
        scanf("%d", &n);
        mx = 0;

        while(n--) {
            scanf("%d", &c);
            if(c > mx) mx = c;
        }

        printf("Case %d: %d\n", tc, mx);
    }

    return 0;
}
